test/Common/TestHelpers.cs

```diff
@@ -111,6 +113,98 @@ public static string GetTaskHubNameFromTestName(string testName, bool enableExte
             return truncatedTestName + deterministicSuffix + randomSuffix;
         }
 
+        /// <summary>
+        /// Returns <c>true</c> if a JSON Durable log has all the minimum-required keys.
+        /// Else, returns <c>false</c>.
+        /// </summary>
+        /// <param name="json">The JSON log to validate.</param>
+        public static bool IsValidJSONLog(JObject json)
+        {
+            List<string> expectedKeys = new List<string>
+            {
+                "EventStampName",
+                "EventPrimaryStampName",
+                "ProviderName",
+                "TaskName",
+                "EventId",
+                "EventTimestamp",
+                "Tenant",
+                "Pid",
+                "Tid",
+            };
+            List<string> keys = json.Properties().Select(p => p.Name).ToList();
```

Comment:
nit: `HashSet<string>` would be more appropriate than `List<string>`. That or just do `json.TryGetValue(expectedKey, out _)` and avoid the extra data structure altogether.